src/modules/dashboard/components/Dialogs/ControlSafeDialog/TransactionTypesSection/shared.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import Avatar from '~core/Avatar';
+
+import { DialogSection } from '~core/Dialog';
+import { SpinnerLoader } from '~core/Preloaders';
+import { AnyUser } from '~data/index';
+import { Address, Message } from '~types/index';
+
+import styles from './TransactionTypesSection.css';
+
+export const AvatarXs = (address: Address, item: AnyUser) => (
```

Comment:
Just my opinion and a nitpick but I feel like the name of this component could be understood better if both letters were capitalized `XS`. For me, when talking about sizes, whether in this context or in the context of clothing, for example, it's more common for both letters to be lower or upper case.

Reply:
Tbh I think this reusable component is not really necessary, but I don't feel strongly about not including it. My only concern is that too many components like this one can make the dev experience a bit messy at times